ground/lib/station.js

```diff
@@ -195,6 +194,10 @@ export class Station extends EventEmitter {
     stream.pipe(parser);
   }
 
+  sendMessage(msg){ //modulation through python AFSK 
+    spawn('/bin/bash', [path.join(__dirname, '..', '..', 'lib', 'modulateData.sh'), this.aprsClient.user, msg]);
```

Comment:
nit: we have `rtlfm_demod.sh` and `moduleData.sh`, let's pick lowercase underscore or camelcase and stick with it ;)